services/director-v2/tests/unit/test_modules_dynamic_sidecar_scheduler.py

```diff
@@ -276,6 +276,7 @@ async def test_scheduler_add_remove(
     assert scheduler_data.service_name not in scheduler._scheduler._to_observe
 
 
+@pytest.mark.flaky(max_runs=3)
```

Comment:
Since when did this one started failing. I'll have a look at it the following sprint.

Reply:
Just a comment, I think that we are missing some library that would really rerun this test 3 times. Therefore at the moment, it would not run 3 times. I have it on my list to check.